packages/grafana-data/src/types/config.ts

```diff
@@ -293,6 +293,7 @@ export interface GrafanaConfig {
   rendererDefaultImageHeight: number;
   rendererDefaultImageScale: number;
   dashboardPerformanceMetrics: string[];
+  scopePerformanceMetrics: boolean;
```

Comment:
@mdvictor - what do you think about skipping this config? Scopes ain't wildly used, to there's not gonna be a lot of metrics reported anyways. I'd add this when we we would expect this to be enabled on specific instances. Or correlate the enablement of this with dashboardPerformanceMetrics eventually. For now, I'd skip it.